monix-reactive/shared/src/test/scala/monix/reactive/consumers/ListConsumerSuite.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2014-2020 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.reactive.consumers
+
+import minitest.TestSuite
+import monix.execution.CancelableFuture
+import monix.execution.exceptions.DummyException
+import monix.execution.schedulers.TestScheduler
+import monix.reactive.{Consumer, Observable}
+
+import scala.util.{Failure, Success}
+
+object ListConsumerSuite extends TestSuite[TestScheduler] {
+  def setup(): TestScheduler = TestScheduler()
+  def tearDown(s: TestScheduler): Unit = {
+    assert(s.state.tasks.isEmpty, "TestScheduler should have no pending tasks")
+  }
+
+  test("should return the same all consumed elements as a sequence") { implicit s =>
```

Comment:
I think it needs `s.tick()` for tests to pass and it's good to merge :)

Reply:
thought it was passing already... 
is there a better way of running minitests than "sbt project reactiveTests" andThen "sbt test"?

Reply:
`ReactiveTests` is something else but I can see how it is confusing :D 

```scala
reactiveJVM/test
```

or

```scala
reactiveJVM/testOnly *NameOfTheTestClass
```